cheatsheet.css

```diff
@@ -103,283 +106,337 @@ hgroup,
 menu,
 nav,
 section {
-  display: block;
+    display: block;
 }
+
 body {
-  line-height: 1;
+    line-height: 1;
 }
+
 blockquote,
 q {
-  quotes: none;
+    quotes: none;
 }
+
 blockquote:before,
 blockquote:after,
 q:before,
 q:after {
-  content: '';
-  content: none;
+    content: '';
+    content: none;
 }
+
 table {
-  border-collapse: collapse;
-  border-spacing: 0;
-  width: 100%;
+    border-collapse: collapse;
+    border-spacing: 0;
+    width: 100%;
 }
+
+
 /******************
  * Custom styling *
  ******************/
+
 body {
-  font-family: 'Ubuntu', sans-serif;
-  text-align: center;
-  color: #2a2a2a;
-  margin: 1em;
+    font-family: 'Ubuntu', sans-serif;
+    text-align: center;
+    color: #2a2a2a;
+    margin: 1em;
 }
+
 h1 {
-  display: table-cell;
-  font-size: 200%;
-  font-weight: bold;
-  vertical-align: middle;
-  text-align: right;
+    display: table-cell;
+    font-size: 200%;
+    font-weight: bold;
+    vertical-align: middle;
+    text-align: right;
 }
+
 h2 {
-  font-size: 1.2em;
-  margin: 8px 0px 8px 20px;
-  font-weight: bold;
+    font-size: 1.2em;
+    margin: 8px 0px 8px 20px;
+    font-weight: bold;
 }
+
 #top h1 {
-  display: block;
-  text-align: right;
+    display: block;
+    text-align: right;
 }
+
 #top h2 {
-  font-family: 'Ubuntu Condensed', sans-serif;
-  font-size: 120%;
-  text-align: right;
-  font-weight: normal;
+    font-family: 'Ubuntu Condensed', sans-serif;
+    font-size: 120%;
+    text-align: right;
+    font-weight: normal;
 }
+
 footer {
-  font-size: 80%;
+    font-size: 80%;
 }
+
 #thanks:before {
-  content: "Thanks (alphabetically): ";
-  font-size: larger;
+    content: "Thanks (alphabetically): ";
+    font-size: larger;
 }
+
 #top {
-  display: inline-flex;
+    display: inline-flex;
 }
+
 #title {
-  margin: auto;
+    margin: auto;
 }
+
 #version {
-  font-size: larger;
-  vertical-align: bottom;
-  margin-left: -1em;
-  margin-top: auto;
-  margin-bottom: 1.5em;
+    font-size: larger;
+    vertical-align: bottom;
+    margin-left: -1em;
+    margin-top: auto;
+    margin-bottom: 1.5em;
 }
+
 #contents {
-  -webkit-columns: 6 34em;
-  -moz-columns: 6 34em;
-  columns: 6 34em;
-  -webkit-column-gap: 0.25em;
-  -moz-column-gap: 0.25em;
-  column-gap: 0.25em;
+    -webkit-columns: 6 34em;
+    -moz-columns: 6 34em;
+    columns: 6 34em;
+    -webkit-column-gap: 0.25em;
+    -moz-column-gap: 0.25em;
+    column-gap: 0.25em;
 }
+
+
```

Comment:
The content below is just a multi-line comment, not a section marker like the others where you added double blank lines above. Maybe this could be changed to be a single blank line?